docs/sources/features/explore/index.md

```diff
@@ -69,9 +108,9 @@ The autocomplete menu can be trigger by pressing Ctrl+Space. The Autocomplete me
 
 Suggestions can appear under the query field - click on them to update your query with the suggested change.
 
-* For counters (monotonically increasing metrics), a rate function will be suggested.
-* For buckets, a histogram function will be suggested.
-* For recording rules, possible to expand the rules.
+- For counters (monotonically increasing metrics), a rate function will be suggested.
```

Comment:
Again, passive voice. Call out who or what is doing the suggesting, or organize this information in a table.

Reply:
Everything below this is just Prettier changing  asterisks to hyphens and is not relevant to Rich history.